src/main/java/com/k3ntako/HTTPServer/ErrorHandler.java

```diff
@@ -1,26 +1,31 @@
 package com.k3ntako.HTTPServer;
 
 public class ErrorHandler implements ErrorHandlerInterface {
-  @Override
-  public Response handleError(HTTPError e){
-    return generateErrorResponse(e.getStatus(), e);
-  }
```

Comment:
I had hoped to take advantage of Java's overloading, but there seems to be an issue when the arguments are the same base type. I just wanted to note this somewhere.

https://stackoverflow.com/questions/1998544/method-has-the-same-erasure-as-another-method-in-type/8467804#8467804

https://docs.oracle.com/javase/tutorial/java/generics/erasure.html